docker/Dockerfile.customopbuilder.clean

```diff
@@ -0,0 +1,92 @@
+FROM tensorflow/tensorflow:custom-op-gpu as base
```

Comment:
As this one is inside `docker` path then we should move all other docker files here as well.

Reply:
ok

Reply:
But this is for another PR, definitely.